src/Utils.ts

```diff
@@ -15,7 +15,8 @@ export function fromEditorConfig(
 	defaults: TextEditorOptions
 ): TextEditorOptions {
 	const resolved: TextEditorOptions = {
-		tabSize: (config.indent_style === 'tab'
+		tabSize: (config.indent_style === 'unset'
+			? 'auto' : config.indent_style === 'tab'
```

Comment:
There is one remaining concern I have about this implementation. Suppose the `.editorconfig` file has `indent_style = unset` but the user/workspace settings have `"editor.autoIndent": false`. This implementation would effectively override that user/workspace setting and it makes zero sense that a value of `unset` would do such a thing, as `unset` should really just defer to the user/workspace settings.

Can you test this scenario and report back as to whether my fears are true?

Reply:
thanks, that's a very valid point and while editor.autoIndent: false behaviour(1) is unaffected by this, a related setting editor.detectIndentation: false is broken by this change. 
I guess here unset should preserve configured value. I'm not sure how exactly to go about this yet, but will investigate soon! 

1. pasting 4 line spaced text into a (detected) 2 line space document; the pasted section stays as 4 line spaced text.

thanks again!

Reply:
I think the fallback with any of these settings should be `workspaceSettings[nameOfSetting] || userSettings[nameOfSetting] || defaultSettings[nameOfSetting]`.

Reply:
Hi, 

Thanks! Think I've got this sorted, albeit with a bit more code than I'd have liked.

First,I had to load the default workspace settings on the initial getDefaultSettings().
Second, I verified workspaceConfig lookup respects those precedence rules.
Finally, broke out setting vscode props to 'auto' into a stricter function.

Notably, indent_size = tab and any implicit sizing info is now also respected. 
The array of unit tests was amended to take into account 'auto'.

If there is anything I can do to rebase or rewrite, please let me know! 
(perhaps irrelevantly, c82f48a, 5a5e705 were to demonstrate thought pattern, rather than be perfect). 

thanks again Jed!